src/workerd/jsg/rtti.capnp

```diff
@@ -156,6 +141,32 @@ struct FunctionType {
   args @1 :List(Type);
 }
 
+struct JsgImplType {
+  # one of the internal jsg types that are not exposed directly but handled specially
+
+  enum Type {
+    v8Isolate @0;
+    # v8::Isolate
```

Comment:
These comments are a bit tautological. They can likely be removed unless there's more that can be said about them